src/styles/sfds/index.scss

```diff
@@ -146,7 +146,7 @@
     }
 
     a {
-      color: $theme-aurora-base;
+      color: var(--link);
```

Comment:
Question: Will this end up being `theme-ultraviolet-base` instead of `theme-aurora-base`? Or is there some other thing that makes this var know it should be `theme-aurora-base`? Maybe we need a `--link-alt` or something?

Reply:
great catch! updated